src/themes/tokens.css

```diff
@@ -1206,11 +1258,12 @@ sl-select::part(listbox) {
 /* Option styling */
 /** ********** **/
 sl-option::part(base) {
+  color: var(--ts-color-text-default);
   padding: var(--sl-spacing-x-small) var(--sl-spacing-2x-small);
 }
 
-sl-option::part(base):not(.option--selected),
-sl-option::part(base):not(.option--current) {
+sl-option[tabindex="0"]::part(base) {
```

Comment:
Question about this selector: I had originally put these `:not` selectors in here to account for the blue background color on the selected items when you focus away from the select (which you can see if you inspect one of the options, since that takes focus away). In that case, we don't want dark text on the dark blue. Does this `tabindex` selector do the same thing?

Reply:
Yes, but not exactly the same -- I also updated the style to not have the blue background. So in the original override for the older SL version, we didn't want the current (focused) item to have the dark blue background & that was working with the older version, but the behavior I was seeing locally (and still see on the current live docsite) with the new version is that when you click on any item on the list with your mouse, it makes the currently selected item in the list (which could be another item on the list), get the dark blue background, which feels odd/buggy, because an item that you're not hovered over or clicking on changes background color, if that makes sense? And initially when I tried just removing the blue background here, with the decs under these `:not` selectors, I was still not able to get rid of the blue background. But this `tabindex` selector works to give only the currently focused item on the list the same gray background that you get on mouse hover, so that was the reason for the change.